docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -254,6 +254,14 @@ Here's an example update when using the `DotNetCoreCLI` task in Azure DevOps:
 +    arguments: '--configuration Release -p:TestingPlatformCommandLineArguments="--report-trx --results-directory $(Agent.TempDirectory) --coverage"'
 ```
 
+## Known limitations
+
+The MSTest SDK has limited integration with NuGet, meaning that the usual NuGet update and Visual Studio UI for managing NuGet packages won't work as expected. See this issue for more details: [NuGet#13127](https://github.com/NuGet/Home/issues/13127).
```

Comment:
```suggestion
The MSTest SDK has limited integration with NuGet, meaning that the usual NuGet update and Visual Studio UI for managing NuGet packages **DOES NOT** work as expected. See this issue for more details: [NuGet#13127](https://github.com/NuGet/Home/issues/13127).
```

or maybe make this whole thing a Warning note?

Reply:
I started with it as warning section above in the file but there was too many notes and warnings making things confusing. Making a warning section here would mean we have 2 notes only, I don't know if that's ok.

I'll doc team suggest what's best.